students/moiseev-vladislav/src/reducers/profileReducer.js

```diff
@@ -1,11 +1,28 @@
-const initialStore = {
-  systemName: "duoxik",
-  firstName: "Vladislav",
-  secondName: "Moiseev",
-  email: "duoxik@gmail.com",
-  description: "Java developer",
-};
+import { LOAD_PROFILE_SUCCESS } from "../actions/profileActions";
+
+const initialStore = {};
 
 export default function profileReducer(store = initialStore, action) {
+  switch (action.type) {
+    case LOAD_PROFILE_SUCCESS: {
+      debugger;
+      const {
+        systemName,
+        firstName,
+        secondName,
+        email,
+        description,
+      } = action.payload;
+
+      return {
```

Comment:
так было бы лаконичней
```suggestion
      return {
        ...store,
        ...action.paylod
      }
```